crates/extensions/c8y_mapper_ext/src/error.rs

```diff
@@ -115,6 +115,9 @@ pub enum ConversionError {
 
     #[error(transparent)]
     FileError(#[from] FileError),
+
+    #[error(transparent)]
+    FileC8yAlarmError(#[from] c8y_api::json_c8y::C8yAlarmError),
```

Comment:
Why a `File` prefix?

Reply:
Lol. I thought it's `From` :D:D

Reply:
Fix with 816a8cb06525fbaef157ef25690ba983ed647949